concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -1525,6 +1526,30 @@ jobs:
                   echo export "${var_name}"=\"$(eval echo \$${var_name})\"
                 done > config/config.sh
 
+      - task: update-kibana-timezone
+        config:
+          platform: linux
+          image: docker:///ruby#2.2-slim
+          params:
+            SYSTEM_DNS_ZONE_NAME: {{system_dns_zone_name}}
+            DEPLOY_ENV: {{deploy_env}}
+          inputs:
+          - name: paas-cf
+          - name: concourse-manifest
+          run:
+            path: sh
+            args:
+            - -e
+            - -c
+            - |
+              VAL_FROM_YAML=$(pwd)/paas-cf/concourse/scripts/val_from_yaml.rb
+              CONCOURSE_ATC_PASSWORD=$("$VAL_FROM_YAML" jobs.concourse.properties.atc.basic_auth_password concourse-manifest/concourse-manifest.yml)
+              export CONCOURSE_ATC_PASSWORD
+              ES_HOST=$(./paas-cf/concourse/scripts/bosh-cli.sh bosh vms 2>&1 | awk -F'|' '$2~/elasticsearch_master/ {print $6}' | head -1)
```

Comment:
I was originally going to say that I don't think we should use `bosh-cli.sh` to loop back through Concourse for tasks like this because it's complex and will make it harder for us to change Concourse authentication in future.

Then I realised that it might be simpler still to use the hostname of the ELB, which points to all of the ES masters and has a basic healthcheck to remove dead nodes:

https://github.com/alphagov/paas-cf/blob/999fd66836efa1fbeaabafa670289e2d475fd4e4/terraform/cloudfoundry/outputs.tf#L81-L83